tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindProgrammesByAncestryAndTypesTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::<public>
+ * @group repository
+ * @group programmesAggregation
+ */
+class FindProgrammesByAncestryAndTypesTest extends AbstractDatabaseTest
+{
+    public function testFindEpisodeGuideChildrenGetCorrectClips()
+    {
+        $this->loadFixtures(['MongrelsFixture']);
+
+        $clipsUnderProgramme = $this->getEntityManager()
+            ->getRepository('ProgrammesPagesService:CoreEntity')
+            ->findProgrammesByAncestryAndType([1, 15], 'Clip', 100, 0);
```

Comment:
yeah this could lead to potentially flaky tests. Take a look at [this](https://github.com/bbc/programmes-pages-service/blob/008ab25f031c3c389dbe6e7fa485952cacef1459/tests/Data/ProgrammesDb/EntityRepository/CollapsedBroadcastRepository/CountByCategoryAncestryAndEndAtDateRangeTest.php#L27). Use that getAncestry function to get the ancestry instead of using the ids straight away